### ISSUE TITLE
Little improvement

### ISSUE THREAD

**Issue by paul-carteron** [feddelegrand7/BARIS#4]
Hello @feddelegrand7 ,
First of all, bravo for this beautiful and surely useful work !! I already tried it a bit and it works perfectly !

I notice that `BARIS_explain()` return a raw character so you have some  : "\n" or "l&#x27". Special character are not read.

An easy way of enhancing it for a better reading is to use `cat` function [here](https://github.com/feddelegrand7/BARIS/blob/008cd7a7f7b579c2671c468eb8b926c235ded0c7/R/BARIS_explain.R#L42) like that :

`return(cat(df_explain$description))`

Have a good day ;)

PS : If you're interested I'm doing quite the same work with IGN data in the [`happign`](https://paul-carteron.github.io/happign/) package

**Comment by feddelegrand7** [comment]
thanks @paul-carteron I'll have a look at Baris_explain, the problem with `cat` is that you cannot assign the result of it to a variable, i.e, this is not possible: x <- cat("hello"). The happign package looks neat, thanks!

**Comment by paul-carteron** [comment]
Yes, you're right. To have both advantages I aready done something like that :

```
foo <- function(text, silent = FALSE){
   if (!silent){
        cat(text)
   } 
   return(text)
}
```

So in both case you assign the text, and you can choose if you wanna display it.
